apps/condo/domains/meter/schema/MeterReading.js

```diff
@@ -41,65 +41,67 @@ const MeterReading = new GQLListSchema('MeterReading', {
             schemaDoc: 'If the meter is single-tariff, then only this value will be filled in;' +
                 'If multi-tariff, then the value of the first tariff will be in this field',
             type: Decimal,
-            hooks: {
-                validateInput: async ({ context, operation, resolvedData, fieldPath, addFieldValidationError }) => {
-                    if (operation === 'create') {
-                        const value = Number(resolvedData[fieldPath])
-                        const lastMeterReading = await getLastBillingAccountMeterReading(context, resolvedData)
-
-                        if (lastMeterReading && lastMeterReading.value1 > value)
-                            addFieldValidationError(`${VALUE_LESS_THAN_PREVIOUS_ERROR}value] Meter reading value less than previous`)
-                    }
-                },
-            },
+            // For now we have decided to refuse the validation of new readings relative to previous ones
+
+            // hooks: {
```

Comment:
I'd rather remove the commencted code from repo, you can still get it from git anyways